spec/helpers/models/time-entry.coffee

```diff
@@ -11,5 +11,7 @@ class TimeEntry extends Model
     task: "tasks"
     user: "users"
 
+  defaultJSONBlacklist: ->
+    ['calculated_attribute']
```

Comment:
Is this the only model with a blacklisted association?

Reply:
This is just checking for a blacklisted attribute -- not necessarily an association.

Can you elaborate at what you're trying to point out?

Reply:
Just wondering if this change is motivated by a feature or was it something that's been broken. Typically, bigmaven API doesn't play well if you send blacklisted attributes over.

Reply:
Exactly. This changes takes the assumption that blacklisted attributes are never sent over to the server.